cmd/create/ocmrole/cmd.go

```diff
@@ -268,6 +268,15 @@ func run(cmd *cobra.Command, argv []string) {
 			os.Exit(1)
 		}
 		if reporter.IsTerminal() {
+			err = checkAdminRoleExists(reporter, awsClient, roleNameRequested, isAdmin)
+			if err != nil {
+				reporter.Errorf(
+					"There was an error checking if the required role name is already an admin role name: %s", err)
+				ocmClient.LogEvent("ROSACreateOCMRoleModeManual", map[string]string{
+					ocm.Response: ocm.Failure,
+				})
+				os.Exit(1)
```

Comment:
@zgalor I wasn't sure if we want to exit here without printing the commands to the user, WDYT?

Reply:
exit